src/model/entities/part.ts

```diff
@@ -69,12 +74,15 @@ export class Part {
    */
   private endState?: PartEndState
 
+  public readonly defaultPart?: IngestedPart
```

Comment:
Seems that the `defaultPart` on the interface has been renamed to `ingestedPart`, unless it has always been named that. 
Should the name of the property on the class be the same name too?

Reply:
Updated